test/e2e/serial/tests/tolerations.go

```diff
@@ -716,6 +726,40 @@ var _ = Describe("[serial][disruptive][rtetols] numaresources RTE tolerations su
 	})
 })
 
+func buildMCPsInfo(cli client.Client, ctx context.Context, nroObj nropv1.NUMAResourcesOperator) ([]mcpInfo, error) {
+	mcpsInfo := []mcpInfo{}
+	var updatedNroObj nropv1.NUMAResourcesOperator
+
+	if err := cli.Get(ctx, client.ObjectKeyFromObject(&nroObj), &updatedNroObj); err != nil {
+		return mcpsInfo, err
+	}
```

Comment:
we should be consistent; either always
`if err := action(); err != nil {`
or always
```
err := action()
if err != nil {
```
I'm biased towards the latter but I don't mind the former if we have at least source-code-level consistency

Reply:
ack, changed that, thanks for pointing out